RNTester/js/AnimatedGratuitousApp/AnExApp.js

```diff
@@ -195,11 +195,6 @@ class Circle extends React.Component<any, any> {
 }
 
 class AnExApp extends React.Component<any, any> {
-  static title = 'Animated - Gratuitous App';
```

Comment:
Many of the example files used static properties to declare the title and description. This didn't work great with flow types and is inconsistent with the other examples so I moved them all to the exports